tools/cpp/BUILD.tpl

```diff
@@ -66,7 +66,7 @@ cc_toolchain(
     linker_files = ":compiler_deps",
     objcopy_files = ":empty",
     strip_files = ":empty",
-    supports_param_files = %{supports_param_files},
```

Comment:
while this substitution is always true here now, we may want to keep this if it's considered public API?

Reply:
I think documentation should show that the default is true, but I think it's fine to leave it too.

Reply:
great going to remove it then for clarity